app/components/funding_widget_component.html.erb

```diff
@@ -30,47 +30,35 @@
-              </p>
-
-              <p>
-                You might be able to get extra funding support <a href="/funding-and-support/if-youre-a-parent-or-carer">if you're a parent or carer</a> and <a href="/funding-and-support/if-youre-disabled">if you're disabled</a>.
-              </p>
-            </div>
-          </div>
-          <div>
-            <h3>Next steps</h3>
-            <div>
-              <p>Use the <a href="https://www.gov.uk/student-finance-calculator">student finance calculator</a> to find out what tuition fee and maintenance loan you could get.<br></p>
-              <p>Explore <a href="/train-to-be-a-teacher">how to train to be a teacher</a>.</p>
+              <p>For this option, a scholarship or bursary is not available. You can still get a <a href="/funding-and-support/tuition-fee-and-maintenance-loans">tuition fee and maintenance loan</a> if you're eligible.</p>
+              <p>You might be able to get extra funding support <a href="/funding-and-support/if-youre-a-parent-or-carer">if you're a parent or carer</a> or <a href="/funding-and-support/if-youre-disabled">if you're disabled</a>.</p>
+              <%= render Content::InsetTextComponent.new(
+                header: "Non-UK citizens:",
+                text: "You're unlikely to get help funding your training unless you have permission to <a href=\"https://www.gov.uk/browse/visas-immigration/settle-in-the-uk\">live permanently in the UK</a>. Find out about <a href=\"/non-uk-teachers/fees-and-funding-for-non-uk-trainees\">funding for non-UK citizens</a>.",
+                color: "purple-white"
+              ) %>
             </div>
           </div>
         <% else %>
           <div class="call-to-action__text">
             <h3><%= sub_head %></h3>
             <div>
               <%= helpers.safe_format(funding_results) %>
-
               <p>You can receive this alongside a <a href="/funding-and-support/tuition-fee-and-maintenance-loans">tuition fee and maintenance loan</a>.</p>
-              <p>You could also get extra funding support <a href="/funding-and-support/if-youre-a-parent-or-carer">if you're a parent or carer</a> and <a href="/funding-and-support/if-youre-disabled">if you're disabled</a>.</p>
+              <p>You could also get extra funding support <a href="/funding-and-support/if-youre-a-parent-or-carer">if you're a parent or carer</a> or <a href="/funding-and-support/if-youre-disabled">if you're disabled</a>.</p>
+              <%= render Content::InsetTextComponent.new(
+                header: "Non-UK citizens:",
+                text: "<p>If you train to teach languages or physics, you may be eligible for a bursary or scholarship. You're unlikely to be eligible for bursaries and scholarships in other subjects.</p>
+                      <br>
```

Comment:
@jenhadfield-dfe Do we need to change the styling here rather than put in `<br>`?

Reply:
I tried to do this, but if i take off the margin on the p tag, it then adds space between the non-UK header making the box much longer. I used the br for now as that was a fix used elsewhere on the site. 
I'm going to create a ticket in the backlog as it's part of a wider smaller fix, where we'd like the content to wrap when there isn't a title. (see image)
<img width="677" alt="Screenshot 2024-08-21 at 12 14 33" src="https://github.com/user-attachments/assets/cffd1974-6b06-4580-89cd-2c5d1e363db8">